userprofile/static/userprofile/extra_logins.js

```diff
@@ -0,0 +1,20 @@
+(function ($) {
+
+  function showExtras(event) {
+    event.preventDefault();
+    const extras = $("#login-box-row").find(".extra-login")
+    extras.show();
+    $(".show-extra-login-btn").hide();
+    extras.find(":input:visible").first().focus() // focus for keyboard navigation
+  }
+
+  $(function () {
+    const row = $("#login-box-row")
+    if (row.find(".login-box").not(".extra-login").length > 0) {
+      row.find(".extra-login").hide();
+      $(".show-extra-login-btn").show();
+      $(".show-extra-login-btn").on("click", showExtras);
```

Comment:
These lines can be joined, which would also mena that the query is not executed twice:

```js
$(".show-extra-login-btn").on("click", showExtras).show();
```